src/ai/defaultai.cc

```diff
@@ -1004,6 +1005,8 @@ void DefaultAI::late_initialization() {
 					verb_log_dbg_time(gametime, "AI %d detected ranger: %s -> %s", player_number(),
 					                  bo.name, lumberjack->name().c_str());
 					bo.set_is(BuildingAttribute::kRanger);
+					rangers_.push_back(bo);
```

Comment:
filling the list wit all rangers and initializing the map with the wood_policies for each ranger